benchmark_utils/generate_config/generate_config_per_dataset.py

```diff
@@ -27,6 +27,9 @@
 
     for filepath in filenames_dataset:
         name = filepath.stem  # Remove the .py suffix
+        if name.startswith('deep'):
```

Comment:
Need to also add:
`if not name.endswith('.py')`

This will ensure with os files like `.DS_STORE`